03-Custom-Actions-In-Python/06-Improving-Performance.ipynb

```diff
@@ -0,0 +1,421 @@
+{
```

Comment:
<p>If you demo this, people <em>will</em> use it for profiling even if you tell them they shouldn't.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Agreed, but this was the easiest way to show performance improvements. I didn't want to look for a package to install, and using `%timeit` on the whole simulation is obviously not representative.

Reply:
<p>I agree with @cbkerr. If a simple %time magic is enough to demonstrate the performance delta of the whole simulation run, then let's use that. The tutorials should cover simple, reusable (copy and paste enabled) code.</p>